Kernel/CommandLine.cpp

```diff
@@ -175,6 +175,11 @@ UNMAP_AFTER_INIT StringView CommandLine::root_device() const
     return lookup("root"sv).value_or("/dev/hda"sv);
 }
 
+bool CommandLine::is_storage_polling_enabled() const
+{
+    return contains("storage_poll"sv);
```

Comment:
I suggest we change this to `nvme_poll` as it seems highly specific to NVMe drives. E.g. if you have a NVMe drive and a regular SATA SSD you probably wouldn't want polling on both.

Reply:
When you put it like that, it makes total sense :) . I guess when we add polling support to SATA, then a similar boot parameter could be introduced. I will fix it up.